src/ToDo.UI/Views/Dialogs/ExpirationDateFlyout.xaml

```diff
@@ -13,8 +13,77 @@
 	  Background="{ThemeResource BackgroundBrush}">
 
 	<Page.Resources>
```

Comment:
Did you generate the code with the proper last version of the Material M3 file ?